Bank_Churn/projects/bank_churn_project_1.py

```diff
@@ -1,51 +1,127 @@
-
 import requests
 import json as json
 import numpy as np
+import argparse
+import tkinter as tk
+import PySimpleGUI as sg
+
+
+# Sample format of data expected
+# {"CreditScore":619,"Age":42,"Tenure":2,"Balance":0,"NumOfProducts":1,"HasCrCard":1,"IsActiveMember":1,"EstimatedSalary":101348.88}
+
+def get_multiple_predictions(values, fields):
+    """
+    This function calculates the predictions on
+    multiple samples
+    Args:
+    values (dict): Dictionary containing url and feature values
+    fields (list): List of strings containing feature names
+    Returns:
+    predictions (list): Contains the list of predictions
+    """
+    values_age = values['Age']
```

Comment:
why are we only doing it for age? Ideally, they should be able to do it for any field right?

Please open a ticket and lets fix it in the next version.